gatsby/content/projects/clients/quadrix.mdx

```diff
@@ -0,0 +1,66 @@
+---
+layout: projectimage
+title: quadrix.chat
+slug: quadrix
+categories:
+ - client
+description: Minimal Matrix client available in all main app stores
+author: Jean-François Alarie
+home: https://quadrix.chat
+language: Typescript
+license: GPL-3.0-or-later
+repo: https://github.com/alariej/quadrix
+room: "#quadrix:matrix.org"
+thumbnail: /docs/projects/images/quadrix-screenshot.png
+screenshot: /docs/projects/images/quadrix-screenshot.png
+maturity: Released
+platforms:
+    - Linux
+    - Mac
+    - Windows
+    - Web
+    - Android
+    - iOS
+    - Mobian
+featured: true
```

Comment:
Tested locally and the `quadrix.chat` project sorts really high compared to the existing Matrix clients in all categories. I know we could mess with `sort_order` but ideally [we don't want to mess with that and set it for everything](https://github.com/matrix-org/matrix.org/pull/1015).

<img src="https://user-images.githubusercontent.com/558581/183513909-3a7d75d9-d376-478d-8e49-48de2f5d1e61.png" width="794">

Reply:
This point may not matter. It does sort last (more as expected) after [fixing the formatting](https://github.com/matrix-org/matrix.org/pull/1348/files#r940673139).